environments/docker.toml

```diff
@@ -273,6 +274,13 @@
         migrations = "db/migrations"
         contracts = ["CDP_MANAGER"]
         rank = "0"
+    [exporter.log_value]
+        path = "transformers/events/log_value/initializer"
+        type = "eth_event"
+        repository = "github.com/makerdao/vdb-mcd-transformers"
+        migrations = "db/migrations"
+        contracts = ["OSM_ETH", "OSM_REP", "OSM_ZRX", "OSM_OMG", "OSM_BAT", "OSM_DGD", "OSM_GNT"]
```

Comment:
Do these contracts correspond to the **PIP**\_* addresses or the **VAL**\_ *addresses in the [changelog](https://changelog.makerdao.com/releases/kovan/0.2.15/contracts.json)? They're mostly the same, except that there is a PIP_SAI address, but not a VAL_SAI one, and I'm wondering if we should include one for Sai.

Reply:
Good question-- it took me a while to realize that the addresses I wanted were even in the changelog, so I actually got them by looking in etherscan and just grabbing every `pip_` value that was passed to the [`Spotter.file`](https://github.com/makerdao/dss/blob/master/src/spot.sol#L77) method. So I guess these addresses are more closely linked with the PIP_* addresses.

I was also curious about the Sai situation, but in the changelog the `PIP_SAI` address is `0x0000000000000000000000000000000000000001` (not a real address ofc), and interestingly that's also what's passed to `Spotter.file`. So I assume there's not really an oracle in the system for Sai.

Reply:
Yeah iirc we just assume sai/dai always == $1. Not sure why an address would be declared for its spotter, though, if that's the case... 🤔